src/amo/reducers/landing.js

```diff
@@ -0,0 +1,36 @@
+import {
+  LANDING_GET,
+  LANDING_LOADED,
+  LANDING_FAILED,
+} from 'core/constants';
+
+
+export const initialState = {
```

Comment:
is `entities` used? I see that some actions are setting it and some tests are checking for it. If so, a default value (even if it's `null`) would help me understand the data structure better. Is `entities` really necessary though if we already have `results` arrays?